docs/example.ipynb

```diff
@@ -186,7 +186,7 @@
    "source": [
     "## Create correlation matrix\n",
     "\n",
-    "We can create a generic correlation matrix and a longer form one for all numerical variables in a data frame using the `corr_matrix` function."
+    "We can create a generic correlation matrix and a longer form one for all numerical variables in a data frame using the `corr_matrix` function. Use `[0]` to get a correlation matrix in its longer form and use `[0]` for a generic correlation matrix."
```

Comment:
Hey Nick! Both options say that you should pick [0]. Perhaps, one of them should be [1]?

Reply:
Yeah, thanks for pointing this out!

Reply:
Looks good!